### ISSUE TITLE
Python Lists vs NumPy Arrays?

### ISSUE THREAD

**Issue by dragonejt** [nasa/OnAIR#110]
Is it worth switching from Python Lists to NumPy arrays as the basis for frames? OnAIR already seems to use NumPy in a lot of places, and NumPy arrays are more static and have more requirements than Python lists, such as the elements all needing to be all of the same type (floats). NumPy arrays are also based in C, which should allow for faster manipulations than with normal lists.

**Comment by khurrameycon** [comment]
Yes, numpy arrays support vectorized operations thus performance increases alot while using np.arrays instead of conventional lists.
